ext/date/php_date_consts.h

```diff
@@ -0,0 +1,32 @@
+/* This is a generated file, edit the .stub.php file instead.
+ * Stub hash: 1fdd521d2e9aea42609a8b297aa8fe49af9d94f9 */
+
+#define DATE_FORMAT_RFC3339 "Y-m-d\\TH:i:sP"
+#define DATE_FORMAT_COOKIE "l, d-M-Y H:i:s T"
+#define DATE_FORMAT_ISO8601 "Y-m-d\\TH:i:sO"
+#define DATE_FORMAT_RFC822 "D, d M y H:i:s O"
+#define DATE_FORMAT_RFC850 "l, d-M-y H:i:s T"
+#define DATE_FORMAT_RFC1036 "D, d M y H:i:s O"
+#define DATE_FORMAT_RFC1123 "D, d M Y H:i:s O"
+#define DATE_FORMAT_RFC7231 "D, d M Y H:i:s \\G\\M\\T"
+#define DATE_FORMAT_RFC2822 "D, d M Y H:i:s O"
+#define DATE_FORMAT_RFC3339 "Y-m-d\\TH:i:sP"
+#define DATE_FORMAT_RFC3339_EXTENDED "Y-m-d\\TH:i:s.vP"
+#define SUNFUNCS_RET_TIMESTAMP 0
+#define SUNFUNCS_RET_STRING 1
+#define SUNFUNCS_RET_DOUBLE 2
+#define PHP_DATE_TIMEZONE_GROUP_AFRICA 1
```

Comment:
You use `0x` hex values in the generated code, but integers here. Should that not be consistent? You're generating this after all.